pkg/operator/reconcile.go

```diff
@@ -55,7 +55,29 @@ func ReconcileConsole(cr *v1alpha1.Console) error {
 	if _, _, err := ApplyOAuth(cr, rt); err != nil {
 		return err
 	}
-
 	return nil
+}
+
+func DeleteAllResources(cr *v1alpha1.Console) error {
+	if err := DeleteService(cr); err != nil {
+		return err
+	}
+	if err := DeleteRoute(cr); err != nil {
+		return err
+	}
+	if err := DeleteConfigMap(cr); err != nil {
+		return err
+	}
+	if err := DeleteDeployment(cr); err != nil {
+		return err
+	}
+	if err := DeleteOAuthSecret(cr); err != nil {
```

Comment:
Remove, you did not create, you do not delete.

Reply:
I do create this secret.  I do not create the client.